Wikipedia/assets/styleoverrides.css

```diff
@@ -1,4 +1 @@
6dHlwZT0iTVNMYXllckdyb3VwIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSg2ODIuMDAwMDAwLCA3NDEuMDAwMDAwKSI+CiAgICAgICAgICAgICAgICA8cGF0aCBkPSJNMjEuODMyNDc3NywxLjY2ODU3MjY5IEwyMS44MDA1OTQ3LDEuNjM2Njg5NyBMOS40NjgyOTI5OCwxMy45Njg5OTE0IEwxMC4wNzI5MDk5LDE0LjU3MzYwODMgTDIyLjM2NDE5OTMsMi4yODIzMTg4NSBMMjQuNDU5Njc2MSw0LjY0NzgyNTc4IEwxMi4xODEyNDM4LDE2LjkyNjI1OCBMMTIuNzg1ODYwNywxNy41MzA4NzQ5IEwyNS4wMjc2OTQ1LDUuMjg5MDQxMDMgTDI1LjA4MDkxNzgsNS4zNDkxMjI4NCBMMjYuMjA0ODU2OSw0LjI2Mzc0NzQ2IEMyNi44Mjk5MTksMy42NjAxMzIwMyAyNi44NDMxNjUxLDIuNjc1NDg2NDUgMjYuMjQ0MTA4OSwyLjA1NTE0NTYgTDI1LjA0Mzk1NTUsMC44MTIzNTAzMjQgQzI0LjQ0MjAzOTUsMC4xODkwNDgwNDYgMjMuNDQ4MTQ3OCwwLjE2NDgzNDQyNyAyMi44MTc2NDg4LDAuNzUxNjUzMTY2IEwyMS44MzI0Nzc3LDEuNjY4NTcyNjkgWiBNOS4zMTk0NzcwMiwxNC4wNDgzMzE4IEw3Ljk0ODU3ODUxLDE5LjA5NzcxMSBMMTIuODQ4MjA3OSwxNy42NjEyMjYzIEw5LjMxOTQ3NzAyLDE0LjA0ODMzMTggWiIgaWQ9ImVkaXQtY29weS01IiBmaWxsPSIjQzNDNkNCIiBza2V0Y2g6dHlwZT0iTVNTaGFwZUdyb3VwIj48L3BhdGg+CiAgICAgICAgICAgICAgICA8cmVjdCBpZD0ibGVmdCIgZmlsbD0iI0MzQzZDQiIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCIgeD0iMCIgeT0iNiIgd2lkdGg9IjEiIGhlaWdodD0iMjEiPjwvcmVjdD4KICAgICAgICAgICAgICAgIDxyZWN0IGlkPSJSZWN0YW5nbGUtMjAiIGZpbGw9IiNDM0M2Q0IiIHNrZXRjaDp0eXBlPSJNU1NoYXBlR3JvdXAiIHg9IjAiIHk9IjI2IiB3aWR0aD0iMjAiIGhlaWdodD0iMSI+PC9yZWN0PgogICAgICAgICAgICAgICAgPHJlY3QgaWQ9IlJlY3RhbmdsZS0yMSIgZmlsbD0iI0MzQzZDQiIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCIgeD0iMTkiIHk9IjE1IiB3aWR0aD0iMSIgaGVpZ2h0PSIxMiI+PC9yZWN0PgogICAgICAgICAgICAgICAgPHJlY3QgaWQ9IlJlY3RhbmdsZS0yMiIgZmlsbD0iI0MzQzZDQiIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCIgeD0iMCIgeT0iNSIgd2lkdGg9IjEyIiBoZWlnaHQ9IjEiPjwvcmVjdD4KICAgICAgICAgICAgICAgIDxnIGlkPSJHcm91cC0zIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxNi4wMDAwMDAsIDE1LjAwMDAwMCkiIHNrZXRjaDp0eXBlPSJNU1NoYXBlR3JvdXAiPgogICAgICAgICAgICAgICAgICAgIDxyZWN0IGlkPSIwIiBzdHJva2U9IiM3Mjc3N0IiIGZpbGw9IiNGRkZGRkYiIHg9IjAuOTMzMzMzMzMzIiB5PSIwIiB3aWR0aD0iNS4xMzMzMzMzMyIgaGVpZ2h0PSI4LjQiIHJ4PSI2Ij48L3JlY3Q+CiAgICAgICAgICAgICAgICAgICAgPHJlY3QgaWQ9IlJlY3RhbmdsZS0zMSIgZmlsbD0iIzhBOTA5NyIgeD0iMCIgeT0iMyIgd2lkdGg9IjciIGhlaWdodD0iNiIgcng9IjEiPjwvcmVjdD4KICAgICAgICAgICAgICAgIDwvZz4KICAgICAgICAgICAgPC9nPgogICAgICAgIDwvZz4KICAgIDwvZz4KPC9zdmc+)}body{min-height:50vh !important}sup.reference a::before{content:" "}sup.reference a::after{content:" "}.reference_highlight a{color:#000000}.quotebox{float:none !important}.wideImageOverride{display:block;margin-left:auto;margin-right:auto}@media (orientation:portrait){.wideImageOverride{width:100% !important;max-width:100% !important;height:auto !important}}@media (orientation:portrait) and (min-device-width:768px){.wideImageOverride{width:60% !important;max-width:60% !important;height:auto !important}}@media (orientation:landscape){.wideImageOverride{width:50% !important;max-width:50% !important;height:auto !important}}.thumbinner,.gallerytext{text-align:center}.thumbcaption{margin-bottom:1.5em !important}li.gallerybox div.thumb>div{margin:0 !important}div.gallerytext>p{margin-top:0 !important}.thumbimage{height:auto !important}.tsingle{max-width:100% !important}
```

Comment:
Xcode's diff viewer made it easier to see the differences than github's does. Changes were pretty minimal and I tested a bit and everything seems to still work and look the same.